### ISSUE TITLE
getValue() for datetime-local integer for unchanged values

### ISSUE THREAD

**Issue by akwebapps** [json-editor/json-editor#331]
### General information
When populating a datetime-local integer value with existing integer value the flatpickr is populated but getValue() pulls an empty string. 

### Steps to reproduce the behavior
Using the editor I generated the initial data and saved it. The json looks like this (working great):
`[
  {
    "title": "My First Session",
    "abstract": "Some cool abstract",
    "sessionTime": 1556212200
  }
]`

When I populate the editor with this json, the input field is correctly displaying the value (working as expected). However, without making any changes, the result of getValue is this:
`[
  {
    "title": "My First Session",
    "abstract": "Some cool abstract"
  }
]`

I dug into the code a bit and spit out what the getValue function was returning for this input type. this.value returns an empty string.

**Comment by pmk65** [comment]
Probably related to #324 
Im working on a fix, but I haven't had time to test it. https://github.com/pmk65/json-editor/tree/datefix

**Comment by peekieisabear** [comment]
Here's a demo:
[http://dev.akwebapps.com/jsonEditor/](url)

Schema:
```
{
	"sessions" : {
	
		"title": "Design Sessions",
		"type": "array",
		"format": "tabs",
		"items": {
			"type": "object",
			"title": "Session",
			"format": "grid",
			"headerTemplate": "{{ self.title }}",
			"properties": {
				"title": {"type": "string","title":"Session Title","options": {"grid_columns": 6}},
				"register": {"type": "string","title":"Registration URL","format":"url","options": {"grid_columns": 6}},
				"abstract": {"type": "string","title":"Brief Abstract","format":"textarea","options":{"grid_columns": 12}},
				"sessionTime": {"type": "integer","title":"Date/Time","format":"datetime-local","options": {
					"grid_columns": 3,
		            "inputAttributes": {"placeholder":"Enter a date/time"},
					"flatpickr": {"time_12hr": true,"inline": true,}
					}
				},
				"experts":{
					"type": "array",
					"title": "Experts",
					"uniqueItems": true,
					"format": "table",
					"options":{"grid_columns": 4},
					"items": {
						"title": "Expert",
					    "type": "string"
					  }
				},
				"topics": {
					"type": "array",
					"title":"Topics",
					"uniqueItems": true,
					"options":{"grid_columns": 5},
					"format": "table",
					"items": {
						"title": "Topics",
					    "type": "string"
					}
				}
			}
		}
	}
}
```
Data:
```
{
	"sessions": [{
		"title": "Today's Session",
		"register": "",
		"abstract": "Some cool abstract which is pretty long for \"brief\". Lorem ipsum dolor sit amet, consectetaur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. \n\nExcepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum Et harumd und lookum like Greek to me, dereud facilis est er expedit distinct. Nam liber te conscient to factor tum poen legum odioque civiuda. Et tam neque pecun modut est neque nonor et imper ned libidig met, consectetur adipiscing elit, sed ut labore et dolore magna aliquam makes one wonder who would ever read this stuff? Bis nostrud exercitation ullam mmodo consequet. Duis aute in voluptate velit esse cillum dolore eu fugiat nulla pariatur. ",
		"sessionTime": 1554780600,
		"experts": ["Ashley Salinas", "Daniel Coleman"],
		"topics": ["Client Solutions & VDI", "Internet of Things", "Networking", "Server Platforms & Management"]
	}, {
		"title": "Tomorrow's Session Name",
		"register": "",
		"abstract": "some abstract is really cool. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum Et harumd und lookum like Greek to me, dereud facilis est er expedit distinct. Nam liber te conscient to factor tum poen legum odioque civiuda. Et tam neque pecun modut est neque nonor et imper ned libidig met, consectetur adipiscing elit, sed ut labore et dolore magna aliquam makes one wonder who would ever read this stuff? Bis nostrud exercitation ullam mmodo consequet. Duis aute in voluptate velit esse cillum dolore eu fugiat nulla pariatur.  ",
		"sessionTime": 1554822600,
		"experts": ["Ashley Salinas", "Ryan Parman", "Daniel Coleman"],
		"topics": ["Client Solutions & VDI", "Cloud Native ", "Internet of Things", "Security & Compliance"]
	}, {
		"title": "Later in the week session title",
		"register": "",
		"abstract": "Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. \n\nExcepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum Et harumd und lookum like Greek to me, dereud facilis est er expedit distinct. Nam liber te conscient to factor tum poen legum odioque civiuda.",
		"sessionTime": 1555008300,
		"experts": ["5 Person", "Ryan Parman", "7 Person"],
		"topics": ["Cloud Native ", "Internet of Things", "Security & Compliance", "Storage Automation & Management"]
	}]
}
```

**Comment by pmk65** [comment]
@akwebapps Could you try testing the current version? (Custom build, not the CDN version)

**Comment by pmk65** [comment]
Since there's no feedback, I assume this was fixed in #350 
So I'm closing this issue.
